index.js

```diff
@@ -28,19 +29,20 @@ module.exports = class LogDNATransport extends Transport {
   }
 
   log(info, callback) {
+    const level = levelTranslate.get(info.level)
+    
     if (info.error instanceof Error) {
       info.error = info.error.stack || info.error.toString()
     }
 
     if (!info.message) {
       // Send the incoming object payload as the message
-      const level = levelTranslate.get(info.level)
       this.logger.log(info, level)
       callback(null, true)
       return
     }
 
-    const {level, message, indexMeta, timestamp, ...meta} = info
+    const {level: ignored, message, indexMeta, timestamp, ...meta} = info
```

Comment:
For clarity, we'd prefer the variable name to be `_` rather than `ignored`, since "ignored" could still some sort of functionality.